test/fixtures/mson-mixin-nonexistent.apib

```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```

Comment:
Hmm interesting
this: 
```
# <API name>
<API description>

# Data Structures

## User (object)
- id
- Include XXX 
```

Will not parse `Include XXX`

While this:
```
# <API name>
<API description>

# Data Structures

## User (object)

comment 

## Properties
- id
- Include XXX
```
does. 

(both cases returns error)